frontend/src/components/Results/Results.js

```diff
@@ -0,0 +1,61 @@
+import "./Results.scss";
+import React, { useEffect, useState, useContext } from "react";
+import { Button } from "@material-ui/core";
+import { useHistory, useLocation } from "react-router";
+import useWindowSize from "react-use/lib/useWindowSize";
+import Confetti from "react-confetti";
+import LoggedContext from "../../contexts/LoggedContext";
+
+const Result = () => {
+  const history = useHistory();
+  const isLogged = useContext(LoggedContext);
+  const location = useLocation();
+  const [place, setPlace] = useState();
+  const { width, height } = useWindowSize();
+
+  useEffect(() => {
+    if (!isLogged) {
+      history.replace("/login");
+    }
+    if (!location.state) {
+      history.replace("/menu");
+    } else {
+      setPlace(location.state.place);
+    }
+  }, [isLogged, history, location.state]);
+
+  const renderConfetti = () => {
+    return place === 1 ? <Confetti width={width} height={height} /> : null;
```

Comment:
You can take advantage of short-circuiting.
`return place && <Confetti width={width} height={height} />;`